CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,31 @@
+•  Be patient
+
+Not everyone has the same level of expertise in language or coding, so be patient if an explanation needs to be given.
+
+•  Be welcoming
+
+Aim to welcome and support people of all backgrounds and identities, of any race, ethnicity, culture, national origin, color, immigration status, social and economic class, educational level, sex, sexual orientation, gender identity and expression, age, size, family status, political belief, religion, and mental and physical ability.
+
+•  Be respectful and friendly
+
+Respect each other and each other's point of view. Different people have different perspectives on issues. Frustration cannot turn into a personal attack. It is important to remember that a community where people feel uncomfortable or threatened is not a productive one. Also remember that with a world-wide community, English may not be the home language of everyone and since all communication happens via text, it's impossible to read what mood the other is in.
+
+•  Be collaborative
+
+BOINC is a complex whole made of many parts, it is the sum of many dreams. Working together between teams that each have their own goal and vision is essential; for the whole to be more than the sum of its parts, each part must make an effort to understand the whole.
+
+•  Ask for help when unsure
+
+Nobody is expected to be perfect in this community. Asking questions early avoids many problems later, so questions are encouraged. Those who answer should be responsive and helpful.
+
+•  Value decisiveness, clarity and consensus
+
+Disagreements, social and technical, are normal, but let's not allow them to persist and fester, leaving others uncertain of the agreed direction. The strength of the BOINC community comes from its diversity, people from a wide range of backgrounds. Being unable to understand why someone holds a viewpoint doesn't mean that that viewpoint is wrong.
+
+•  Resolve issues
+
+Don't forget that it is human to err and that blaming each other isn't productive. Instead, focus on helping to resolve issues and learning from mistakes. Participants to the project should try to resolve disagreements constructively. When it is impossible to reach an agreement, follow the [process outlined in the Governance document](https://github.com/BOINC/boinc-policy/blob/master/Governance.md#5-decision-processes)to come to a solution.

```

Comment:
Should the start read like this instead?  Don't forget that to err is human and...

https://en.wiktionary.org/wiki/to_err_is_human

Reply:
If you don't mind, I keep it like this. It was like that in the original text I took parts of the CoC from at [the ToDoGroup](http://todogroup.org/opencodeofconduct/) (in the Try to understand why we disagree section).

Reply:
I don't mind at all.  I could say this can be closed.